kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_complex_variables.bpmn

```diff
@@ -13,13 +13,13 @@
     <bpmn:userTask id="UserTask_1kfz8x" name="UserTask1" camunda:assignee="admin">
       <bpmn:extensionElements>
         <camunda:properties>
-          <camunda:property name="taskana.classification-key" value="L1050" />
+          <camunda:property name="kadai.classification-key" value="L1050" />
         </camunda:properties>
       </bpmn:extensionElements>
       <bpmn:incoming>SequenceFlow_0ncfypl</bpmn:incoming>
       <bpmn:outgoing>SequenceFlow_0ptcaku</bpmn:outgoing>
     </bpmn:userTask>
-    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```

Comment:
io.kadai.impl.ComplexProcessVariableSetter ?

Reply:
You're right, I will change it